lte/gateway/c/session_manager/PipelinedClient.h

```diff
@@ -288,6 +288,8 @@ class AsyncPipelinedClient : public GRPCReceiver, public PipelinedClient {
       const magma::UEMacFlowRequest req, const int retries, Status status,
       FlowResponse resp);
 
+  void poll_stats(int cookie, int cookie_mask);
```

Comment:
add docs on what cookie and cookie mask are. (I have no idea what they are :) )
```
  /**
   * @brief TODO
   *
   * @param cookie // TODO
   * @param cookie_mask // TODO
   */
```

Reply:
You could just go ahead and add `std::function<void(Status, RuleRecordTable)> callback` as a parameter for this function, as you'll need it anyways.